domainservice/src/main/kotlin/systems/ajax/malov/stockanalyzer/kafka/processor/StockPriceNotificationProcessor.kt

```diff
@@ -0,0 +1,36 @@
+package systems.ajax.malov.stockanalyzer.kafka.processor
+
+import jakarta.annotation.PostConstruct
+import org.springframework.stereotype.Component
+import reactor.core.scheduler.Schedulers
+import reactor.kafka.receiver.KafkaReceiver
+import reactor.kotlin.core.publisher.toMono
+import systems.ajax.malov.internalapi.output.pubsub.stock.stock_price.proto.StockPrice
+import systems.ajax.malov.stockanalyzer.kafka.producer.StockPriceNotificationProducer
+import systems.ajax.malov.stockanalyzer.service.NotificationStockPriceService
+
+@Component
+class StockPriceNotificationProcessor(
+    private val kafkaStockPriceConsumer: KafkaReceiver<String, StockPrice>,
+    private val kafkaNotificationStockPriceSender: StockPriceNotificationProducer,
+    private val notificationStockPriceService: NotificationStockPriceService,
+) {
+    @PostConstruct
+    fun subscribeToEvents() {
+        kafkaStockPriceConsumer.receiveAutoAck()
```

Comment:
autoAck may be acceptable for pet projects, but in big applications, it's better to acknowledge records manually